pandas/core/series.py

```diff
@@ -5072,9 +5072,9 @@ def between(self, left, right, inclusive="both") -> Series:
         4    False
         dtype: bool
 
-        With `inclusive` set to ``False`` boundary values are excluded:
+        With `inclusive` set to ``"neither"`` boundary values are excluded:
```

Comment:
hmm, remind me. why did we deprecate True/False as allowable arguments?

Reply:
because T/F is only 2 options. 'both', 'neither' 'left' 'right' is all cases. If working towards standardising this type of arg across methods, makes sense to have all encompassing options (#40245)

Reply:
Thanks @attack68. There was no discussion on the deprecation of the existing parameters in #40245. The issue is labelled an enhancement. We probably should have had a discussion on the deprecation also.

Reply:
the PR (#40628) was initially an expansion which allowed T/F for backwards compat. First mention of deprecation came from jreback [here](https://github.com/pandas-dev/pandas/pull/40628#discussion_r603374738) as part of review process and the changes were requested. I agree with your comment.